storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -540,6 +549,18 @@ func (c *PrometheusConverter) addTimeSeriesIfNeeded(lbls []prompb.Label, startTi
 	}
 }
 
+// handleStartTime adds a zero sample 1 millisecond before ts iff startTs == ts.
```

Comment:
It might be worth mentioning that we're doing this because PRWv1 doesn't support Created Timestamps.

Once we migrate the translation layer to the PRWv2 Protocol, this won't be needed and we'll just let the PRWv2 handler take care of `StartTimeUnixNano`

Reply:
Added to the doc comment, PTAL.

Reply:
Just pushed the commit altering the doc comment, had somehow omitted to push it earlier :facepalm: